proto/prysm/v1alpha1/beacon_chain.proto

```diff
@@ -228,6 +185,7 @@ service BeaconChain {
     // validator attestations. This endpoint allows for retrieval of genesis
     // information via a boolean query filter.
     rpc GetValidatorParticipation(GetValidatorParticipationRequest) returns (ValidatorParticipationResponse) {
+        option deprecated = true;
```

Comment:
Is this newly deprecated?

Reply:
Yes, we had a discussion i wanted to get rid of it but realized we don't have it deprecated yet...